docs/topic-guides/system-architecture.txt

```diff
@@ -30,3 +30,6 @@ like checkpointing model state. The master may then send more workloads
 or terminate the trial runner (freeing the slot). When sending a
 workload to the trial runner, the master consults with the searcher to
 determine which workload to run next.
+
+For more information about the parts of Determined, see
```

Comment:
blocking: Is there a better word here than "parts"?

Reply:
I'll just turn this into a "Next Steps" section and not have any text. It was fluff anyway.